README.md

```diff
@@ -40,6 +40,10 @@ A GUI application which is used to create mods.
 ### FrostyModManager
 A GUI application which is used to select what mods to apply to the game.
 
+### FrostyCLI
+A CLI application which is used to create, update and apply mods. 
+See [Readme](https://github.com/DAPOTTIS/FrostyToolsuite/blob/Readme-Stuff/FrostyCli/README-CLI.md).
```

Comment:
make sure this link is pointing to the correct repo, even if the file currently does not exist